infrastructure/ArmTemplates/DeployWebApp.json

```diff
@@ -0,0 +1,374 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
```

Comment:
The existing one takes care for setting up geneva logging as well. Can you combine that here?

Reply:
I intentionally created this to remove the geneva logging in environments where we don't have geneva set up. I think it should be its own separate ARM template.